src/main/java/frc/robot/RobotContainer.java

```diff
@@ -86,26 +85,35 @@ private void configureButtonBindings() {
         sensitiveDriverControlCommand = new SensitiveDriverControl(driverStick);
 
     }
-    
+
     /**
      * Use this to pass the autonomous command to the main {@link Robot} class.
      *
      * @return the command to run in autonomous
      */
     public Command getAutonomousCommand() {
-        int selectorOne = 0, selectorTwo = 0;
-        if (analogSelectorOne != null) selectorOne = analogSelectorOne.getIndex();
-        if (analogSelectorTwo != null) selectorTwo = analogSelectorTwo.getIndex();
+        int selectorOne = 1, selectorTwo = 1;
+        if (analogSelectorOne != null)
```

Comment:
Nit: Your choice but consider adding {} around 1 line if statements across 2 lines. I know that in Java this is permitted but it is good practice to use brackets so that it becomes more obvious what is grouped with the if and for future additions it is simpler to add lines without cluttering changes for brackets.